manager/controllers/advisories.go

```diff
@@ -188,15 +188,15 @@ func AdvisoriesListIDsHandler(c *gin.Context) {
 func buildQueryAdvisories(db *gorm.DB, account int) *gorm.DB {
 	query := db.Table("advisory_metadata am").
 		Select(AdvisoriesSelect).
-		Joins("JOIN advisory_account_data aad ON am.id = aad.advisory_id and aad.systems_affected > 0").
+		Joins("JOIN advisory_account_data aad ON am.id = aad.advisory_id and aad.systems_installable > 0").
 		Joins("JOIN advisory_type at ON am.advisory_type_id = at.id").
 		Where("aad.rh_account_id = ?", account)
 	return query
 }
 
 func buildAdvisoryAccountDataQuery(db *gorm.DB, account int) *gorm.DB {
 	query := database.SystemAdvisories(db, account).
-		Select("sa.advisory_id, sp.rh_account_id as rh_account_id, 0 as status_id, count(sp.id) as systems_affected").
+		Select("sa.advisory_id, sp.rh_account_id as rh_account_id, count(sp.id) as systems_installable").
```

Comment:
`systems_installable` here is in fact installable + applicable because it is `count(sp.id)`?

Reply:
Answering all at once:
This was actually mean just to update schema and fix / rename other stuff to make it consistent.
Real code changes are going to go in next PRs.